Description: explicitly link against the following libraries to avoid FTBFS
with gold linker or ld --no-add-needed: pthread, dl, boost.system and
boost.filesystem.
Author: Cristian Greco <cristian@regolo.cc>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -43,7 +43,7 @@
 	
 	SET(Boost_USE_MULTITHREAD ON)
 	SET(Boost_USE_STATIC_LIBS OFF)
-	find_package(Boost COMPONENTS date_time REQUIRED)
+	find_package(Boost COMPONENTS date_time system filesystem REQUIRED)
 	
 	if(Boost_FOUND)
 		message(STATUS "boost-datetime found OK")
@@ -427,7 +427,7 @@
 add_executable(fatrat ${fatrat_SRCS} ${fatrat_MOC_SRCS} ${fatrat_UIS_H}
 ${fatrat_QRC_H} ${lrelease_outputs})
 
-target_link_libraries(fatrat ${QT_LIBRARIES} ${libtorrent_LDFLAGS} ${gloox_LDFLAGS} ${curl_LDFLAGS} ${Boost_LIBRARIES} ${QT_ADDITIONAL_LIBS} -export-dynamic)
+target_link_libraries(fatrat ${QT_LIBRARIES} ${libtorrent_LDFLAGS} ${gloox_LDFLAGS} ${curl_LDFLAGS} ${Boost_LIBRARIES} ${QT_ADDITIONAL_LIBS} -lpthread -ldl -export-dynamic)
 
 set(fatrat_DEV_HEADERS
 	${CMAKE_CURRENT_SOURCE_DIR}/src/fatrat.h
